torch/onnx/symbolic_registry.py

```diff
@@ -23,19 +23,32 @@ def register_version(domain, version):
     register_ops_in_version(domain, version)
 
 
+def register_ops_helper(domain, version, iter_version):
+    version_ops = get_ops_in_version(iter_version)
+    for op in version_ops:
+        if isfunction(op[1]) and not is_registered_op(op[0], domain, version):
+            register_op(op[0], op[1], domain, version)
+
+
 def register_ops_in_version(domain, version):
     # iterates through the symbolic functions of
     # the specified opset version, and the previous
     # opset versions for operators supported in
-    # previous versions
+    # previous versions.
+
+    # Opset 9 is the base version.
```

Comment:
add some comment about why we choose opset 9 as base version? and as base version, what will happen. it will happen our developer maintain the code in future.

Reply:
added